modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/AbstractClientService.java

```diff
@@ -178,20 +180,24 @@ public <T extends Message> CompletableFuture<Message> invokeWithDone(final PeerI
     public <T extends Message> CompletableFuture<Message> invokeWithDone(final PeerId peerId, final Message request,
         final RpcResponseClosure<T> done, final int timeoutMs,
         final Executor rpcExecutor) {
-        return invokeWithDone(peerId, request, null, done, timeoutMs, rpcExecutor);
+        return invokeWithDone(peerId, request, null, done, timeoutMs, rpcExecutor, this.rpcClient);
     }
 
     public <T extends Message> CompletableFuture<Message> invokeWithDone(final PeerId peerId, final Message request,
         final InvokeContext ctx,
         final RpcResponseClosure<T> done, final int timeoutMs) {
-        return invokeWithDone(peerId, request, ctx, done, timeoutMs, this.rpcExecutor);
+        return invokeWithDone(peerId, request, ctx, done, timeoutMs, this.rpcExecutor, this.rpcClient);
     }
 
-    public <T extends Message> CompletableFuture<Message> invokeWithDone(final PeerId peerId, final Message request,
-        final InvokeContext ctx,
-        final RpcResponseClosure<T> done, final int timeoutMs,
-        final Executor rpcExecutor) {
-        final RpcClient rc = this.rpcClient;
+    public <T extends Message> CompletableFuture<Message> invokeWithDone(
+            PeerId peerId,
```

Comment:
Avoid reformatting raft code.

Reply:
I applied formatting that is used in our entire project.